lib/rules/check-hbs-template-literals.js

```diff
@@ -88,6 +71,48 @@ module.exports = {
   }
 }
 
+function getResults(options) {
+  let runAsync = async ({hbs, context}) => {
+    let TemplateLinter;
```

Comment:
all this stuff needed to be moved into the function wrapped by forceSync, because something weird happens with scope in there